legal_db/static/app.js

```diff
@@ -1,3 +1,16 @@
+// Allow toggle dropdown of the navbar burger menu
+const burgerMenu = document.getElementsByClassName('navbar-burger')[0];
+burgerMenu.addEventListener('click', () => {
+    const menu = document.getElementsByClassName('navbar-menu')[0];
+    menu.classList.toggle('is-active');
+    burgerMenu.classList.toggle('is-active');
+    if (burgerMenu.classList.contains('is-active')) {
+        burgerMenu.setAttribute('aria-expanded', true);
+    } else {
+        burgerMenu.setAttribute('aria-expanded', false);
+    }
```

Comment:
how about:

`burgerMenu.setAttribute('aria-expanded', burgerMenu.classList.contains('is-active'));`

😸

Reply:
Better! thanks 😄